tests/vmcheck/test.sh

```diff
@@ -148,6 +148,8 @@ for tf in $(find . -name 'test-*.sh' | sort); do
       vm_reboot &>> ${LOG}
     fi
 
+    # make sure to clean up any pending & rollback deployments
+    vm_rpmostree cleanup -b -p -r || :
```

Comment:
Yeah, this type of thing bit me with livefs tests...in particular trying to test modified files in `/etc`, any leftover changes leak across tests.  I basically gave up on fixing it and rerun vagrant each time now.